hypothesis-python/docs/strategies.rst

```diff
@@ -25,7 +25,8 @@ Some packages provide strategies directly:
   `D-Bus signatures <https://dbus.freedesktop.org>`_.
 * :pypi:`hypothesis_sqlalchemy` - strategies to generate :pypi:`SQLAlchemy` objects.
 * :pypi:`hypothesis-ros` - strategies to generate messages and parameters for the `Robot Operating System <http://www.ros.org/>`_.
-
+* :pypi:`hypothesis-csv` - strategy to generate CSV files.
+*
```

Comment:
Docs build is complaining about this bullet; it should be a blank line.

Reply:
can you please tell me how to reproduce the build locally to validate? I couldn't find it in the docs

Reply:
or should I just go through the ci?

Reply:
anyway I pushed the fix